--- xdg-user-dirs-gtk-0.10/user-dirs-update-gtk.desktop.in~	2013-01-21 15:18:32.000000000 +0200
+++ xdg-user-dirs-gtk-0.10/user-dirs-update-gtk.desktop.in	2013-02-23 15:29:11.580080585 +0200
@@ -4,7 +4,7 @@
 _Name=User folders update
 _Comment=Update common folders names to match current locale
 Terminal=false
-OnlyShowIn=GNOME;LXDE;Unity;
+OnlyShowIn=GNOME;LXDE;Unity;MATE;
 Type=Application
 StartupNotify=false
 X-KDE-autostart-after=panel
